py-rse/bash-advanced.Rmd

```diff
@@ -625,10 +625,31 @@ $ find . -name "*.txt"
 ./data/jane_eyre.txt
 ```
 
-As stated earlier,
-the command line's power lies in combining tools.
-We have seen how to do that with pipes;
-let's use another technique to see how large our HTML files are:
+Notice the quotes around `"*.txt"`.
+If we omit them and type:
+
+```shell
+$ find . -name *.txt
+```
+
+then the shell tries to expand the `*` wildcard in `*.txt`
+*before* running `find`.
+Since there aren't any text files in the current directory,
+the expanded list is empty,
+so the shell tries to run the equivalent of
+
+```shell
+$ find . -name
+```
+
+and gives us the error message:
+
+```text
+find: -name: requires additional arguments
+```
+
+We have seen before how to combine commands using pipes.
+let's use another technique to see how large our books are:
```

Comment:
```suggestion
We have seen before how to combine commands using pipes,
let's use another technique to see how large our books are:
```

Reply:
Used capital 'L' to start a new sentence rather than joining them up.